docs/modules/mssqlfg.md

```diff
@@ -0,0 +1,300 @@
+# [Azure SQL Database Failover Group](https://docs.microsoft.com/en-us/azure/sql-database/sql-database-geo-replication-overview)
+
+_Note: this module is PREVIEW. It is under moderate development, but future releases may still break the API._
+
+Open Service Broker for Azure (OSBA) contains five Azure SQL Database Failover Group services. These services enable you to select the most appropriate provisioning scenario for your needs. These services are:
+
+| Service Name | Description |
+|--------------|-------------|
+| `azure-sql-12-0-dr-dbms-pair-registered` | Register two existing servers as a service instance. |
+| `azure-sql-12-0-dr-database-pair` | Provision two new databases which make up a failover group upon a previous DBMS pair. |
+| `azure-sql-12-0-dr-database-pair-registered` | Register a failover group upon a previous DBMS pair as a service instance. |
+| `azure-sql-12-0-dr-database-pair-from-existing` | Taking over an existing failover group (included the databases) upon a previous DBMS pair as a service instance. |
+| `azure-sql-12-0-dr-database-pair-from-existing-primary` | Taking over an existing database upon the primary server of a previous DBMS pair, and extend it to a failover group deployment. |
+
+All the services in this module require `ENABLE_DISASTER_RECOVERY_SERVICES` to be `true` in OSBA environment variables. Besides, `azure-sql-12-0-dr-database-pair-from-existing` and `azure-sql-12-0-dr-database-pair-from-existing-primary` require `ENABLE_MIGRATION_SERVICES` to be `true`. The `azure-sql-12-0-dr-dbms-pair-registered` service allows you to provide an existing primary server and an existing secondary server for the DR deployment. For more information on each service, refer to the descriptions below.
```

Comment:
This sentence: "The `azure-sql-12-0-dr-dbms-pair-registered` service allows you to provide an existing primary server and an existing secondary server for the DR deployment." Why `azure-sql-12-0-dr-dbms-pair-registered` is mentioned and only it is mentioned here?

Reply:
Oh, I should delete it. Thanks.